contribute/editorial_board_guide.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Editorial board guide
+summary: This guide is there to help editors.
+---
+
+## All you need to know about this GitHub repository
+
+### General rules
+
+As an editor, try to work as much as possible on a different branch than the master branch. This can be on the elixir-europe rdm-toolkit repo or your own fork. Open a pull request if you want to merge your changes with the master branch. This way it is possible for other editors to give feedback on your changes. Typos or other small fixes can of course be done immediately on the master branch.
+
+### The google doc way of contributing
+This process is sketched below.
+
+{% include image.html file="googledoc_way_flow.svg" alt="Process of contributing via Google docs" click=true %}
+
+### Overview of the file structure in GitHub
+
+The content of the website is built up using markdown files found in the [root](https://github.com/elixir-europe/infectious-diseases-toolkit) of the repository, spread in subdirectories according to there website section (human_biomolecular_data, human_clinical_and_health_data, contribute, social_and_economic_impact, ...).
+
+### Markdown file naming
+
+- Markdown files should be named without capitals and without spaces (replace them with underscores).
+- Make sure that the markdown file has a unique name.
+- If the markdown file is named *example.md*, the page will be found at https://www.infectious-diseases-toolkit.org/.
```

Comment:
Is the url given correct? Should it be `https://www.infectious-diseases-toolkit.org/example` instead?

Reply:
you are right!

Reply:
but best is actually to give an example with url/section/example like i did the other times